src/smt/proof_logger.h

```diff
@@ -0,0 +1,90 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2024 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Proof logger utility.
+ */
+
+#include "cvc5_private.h"
+
+#ifndef CVC5__SMT__PROOF_LOGGER_H
+#define CVC5__SMT__PROOF_LOGGER_H
+
+#include "proof/proof_node.h"
+#include "smt/env_obj.h"
+
+namespace cvc5::internal {
+
+/**
+ * The purpose of this class is output proofs for all reasoning the solver
+ * does on-the-fly. It is enabled when proof logging is enabled.
+ * 
```

Comment:
```suggestion
 *
```
There are other occurrences. Clang-format should delete them, I think.